fineract-provider/src/main/java/org/apache/fineract/infrastructure/core/domain/FineractPlatformTenantConnection.java

```diff
@@ -171,10 +171,12 @@ public int getMinEvictableIdleTimeMillis() {
         return this.minEvictableIdleTimeMillis;
     }
 
+    @Deprecated
```

Comment:
I think let's just cut this out from here and from the DB as well. We'll mention it in the changelog.

Reply:
Very well... on the way

Reply:
@galovics done... dropped also the columns from the tenant configuration DB.